apps/browser/src/autofill/popup/fido2/fido2.component.ts

```
@@ -385,11 +385,14 @@ export class Fido2Component implements OnInit, OnDestroy {
   }
 
   protected async search() {
+    const userId = await firstValueFrom(this.accountService.activeAccount$.pipe(map((a) => a?.id)));
+
     this.hasSearched = true;
-    const isSearchable = await this.searchService.isSearchable(this.searchText);
+    const isSearchable = await this.searchService.isSearchable(userId, this.searchText);
```

There will always be a `userId` here, right? What will happen if the `userId` is invalid in some way?

---

thanks for this. Updated to apply the `getUserId`. should throw an error if ever null.